exchange/src/randomness/brownian.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file brownian.hpp
+ * @author Andrew Li (andrewli@u.northwestern.edu)
+ * @brief Brownian motion randomness to simulate market chaos
+ * @version 0.1
+ * @date 2024-01-12
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#pragma once
+
+#include <random>
+#include <vector>
+
+namespace nutc {
+
+namespace brownian {
+
+const double BROWNIAN_MOTION_DEVIATION = 0.4;
+
+class BrownianMotion {
+    unsigned int seed;
```

Comment:
i don't see the point of storing this, its only used in initialization

Reply:
It's used in the normal distribution to get a random variable, so it's needed every single time .next() is called